src/creature/creature.cpp

```diff
@@ -39,4 +40,20 @@ void Creature::PrintStats()
 	std::cout << "==============================" << std::endl;
 }
 
-int Creature::get_hp() { return m_hp; }
+int Creature::get_hp() { return m_hp; };
```

Comment:
FYI, it's customary to not have any `;` after function definitions (see `PrintStats` above). Also, we separate function definitions by an empty line.